test/buy-tshirt.spec.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable protractor/no-describe-selectors */
```

Comment:
Remove this comment

Reply:
I left the comment because as I am using eslint, it gives me a warning that is solved by implementing the POM, which is the next step of the original workshop